src/js/common/zoom.js

```diff
@@ -0,0 +1,71 @@
+{
+
+const filter_in_range_data = (args, range) => {
+  const is_data_in_range = (data, range) => {
+    return data > Math.min(range[0], range[1]) && data < Math.max(range[0], range[1]);
+  }
+
+  return d => {
+    const x_in_range = ('x' in range) ? is_data_in_range(d[args.x_accessor], range.x) : true;
```

Comment:
This could be restated more simply as:

```js
const x_in_range = !('x' in range) || is_data_in_range(d[args.x_accessor], range.x);
```

Same with the statement just below.